device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/Twin.java

```diff
@@ -211,7 +213,13 @@ public TwinCollection getReportedProperties()
     public String toString()
     {
         /* SRS_TWIN_STATE_21_008: [The toString shall return a String with the information in this class in a pretty print JSON.] */
```

Comment:
Do you want to continue removing these types of comments?

Reply:
Yes